shell/common/electron_constants.h

```diff
@@ -21,6 +21,23 @@ inline constexpr std::string_view kDeviceVendorIdKey = "vendorId";
 inline constexpr std::string_view kDeviceProductIdKey = "productId";
 inline constexpr std::string_view kDeviceSerialNumberKey = "serialNumber";
 
+// Window state preference keys
+inline constexpr std::string_view kLeft = "left";
+inline constexpr std::string_view kTop = "top";
+inline constexpr std::string_view kRight = "right";
+inline constexpr std::string_view kBottom = "bottom";
+
+inline constexpr std::string_view kMaximized = "maximized";
+inline constexpr std::string_view kFullscreen = "fullscreen";
+inline constexpr std::string_view kKiosk = "kiosk";
+
+inline constexpr std::string_view kWorkAreaLeft = "work_area_left";
+inline constexpr std::string_view kWorkAreaTop = "work_area_top";
+inline constexpr std::string_view kWorkAreaRight = "work_area_right";
+inline constexpr std::string_view kWorkAreaBottom = "work_area_bottom";
```

Comment:
Is there a reason these are snake case, some kind of chromium consistency? If not can we make things camel case please